### PR TITLE
Fix various copyediting issues in concurrency post

### PR DESCRIPTION
**Pull request by zwegner**

### CODE REVIEW

File: _posts/2020-07-06-concurrency-costs.md [hunk 1]
```diff
@@ -451,7 +451,7 @@ class tls_counter {
     /* per-thread tls_counter object */
     static thread_local tls_counter tls;
 
-    /** add ourself to the counter list */
+    /** add ourselves to the counter list */
```
Comment (travisdowns) [inline _posts/2020-07-06-concurrency-costs.md:454]: Maybe this is archaic, but I think it is correct either way? In any case, I considered it when I wrote it... but yeah I'll change it.

File: _posts/2020-07-06-concurrency-costs.md [hunk 2]
```diff
@@ -557,15 +557,15 @@ The _~Cost_ column is a _very_ approximate estimate of the cost of each "occurre
...
 
 The _Key Characteristic_ tries to get across the idea of the level in one bit-sized chunk.
 
 | Level |  Name | ~Cost (ns) | Perf Event | Local | Key Characteristic |
 | ---   | ------| -------------:|:----------------:|:----:| -- |
 | 0 | Vanilla | low | depends | **Yes**{:.yes.yesno} | No atomic instructions or contended accesses at all |
 | 1 | Uncontended Atomic | 10 | `mem_inst_retired.` `lock_loads` | **Yes**{:.yes.yesno} | Atomic instructions without contention |
-| 2 | True Sharing | 40 - 400 | `mem_load_l3_hit_retired.` `xsnp_hitm` | **No**{:.no.yesno} | Contented atomics or locks |
+| 2 | True Sharing | 40 - 400 | `mem_load_l3_hit_retired.` `xsnp_hitm` | **No**{:.no.yesno} | Contended atomics or locks |
```
Comment (travisdowns) [inline _posts/2020-07-06-concurrency-costs.md:568]: lol

File: _posts/2020-07-06-concurrency-costs.md [hunk 3]
```diff
@@ -588,7 +588,7 @@ Here's a quick look at some usual and unusual ways of achieving levels lower on
...
+You probably don't want to really be in level 4 but it's certainly better than level 5. So, if you still have your job and your users haven't all abandoned you, it's usually pretty easy to get out of level 5. More than half the battle is just recognizing what's going on and from there the solution is often clear. Many times, you've simply violated some rule like "don't use pure spinlocks in userspace" or "you built a spinlock by accident" or "so-and-so accidentally held that core lock during IO". There's almost never any inherent reason you'd need to stay in level 5 and you can usually find an almost tradeoff-free fix.
```
Comment (travisdowns) [inline _posts/2020-07-06-concurrency-costs.md:591]: for whatever reason GitHub doesn't propery highlght the removal of duplicate words? Very annoying:

image

Normally the removed part is highlighted but not in these cases for some reason.

**Comment by travisdowns** [comment]
Thanks a ton!

File: _posts/2020-07-06-concurrency-costs.md (hunk 1 above)
Comment (zwegner) [reply _posts/2020-07-06-concurrency-costs.md:454]: Maybe so? The singular/plural mixing of the word "ourself" makes it sound weird to me in general, but if I think about the royal "our" then it makes more sense.

File: _posts/2020-07-06-concurrency-costs.md (hunk 2 above)
Comment (zwegner) [reply _posts/2020-07-06-concurrency-costs.md:568]: Yeah that one made me smile. We should all strive to write contented software!

File: _posts/2020-07-06-concurrency-costs.md (hunk 3 above)
Comment (zwegner) [reply _posts/2020-07-06-concurrency-costs.md:591]: Yeah I noticed that too, pretty irritating. Maybe it's the line length?

Comment (travisdowns) [reply _posts/2020-07-06-concurrency-costs.md:591]: > Maybe it's the line length?

Yeah could be. I couldn't find a pattern. I just gave up when it didn't show and trusted you at some point. `git diff` after pulling down the changes works fine though.

**Pull request merged**